app/javascript/mastodon/initial_state.js

```diff
@@ -132,5 +133,6 @@ export const usePendingItems = getMeta('use_pending_items');
 export const version = getMeta('version');
 export const translationEnabled = getMeta('translation_enabled');
 export const languages = initialState?.languages;
+export const publish_button_text = getMeta('publish_button_text');
```

Comment:
Should be lowerCamelCase like all other properties.

Reply:
Not sure what happened here, thanks.